drizzlepac/hlautils/photometry_tools.py

```diff
@@ -141,8 +141,8 @@ def iraf_style_photometry(phot_apertures, bg_apertures, data, platescale,
         else:
             flux_error = compute_phot_error(flux, bg_phot, bg_method, ap_area, epadu)
 
-        mag = zero_point - 2.5 * np.log10(flux)
-        mag_err = 1.0857 * flux_error / flux
+        mag = convert_flux_to_abmag(flux, photflam, photplam)
+        mag_err = 1.0857 * flux_error / flux # TODO: Doublecheck this
```

Comment:
It would be helpful to document where this computation came from, especially the 'magic number' 1.0857. It can be done in some later PR.

Reply:
Added comment with brief explanation and link to further reading in commit [b74f64d](https://github.com/spacetelescope/drizzlepac/pull/481/commits/b74f64dbff7f595f20ae1831eaa16fba15a170c3)